receiver/journaldreceiver/metadata.yaml

```diff
@@ -7,3 +7,8 @@ status:
   distributions: [contrib, splunk, observiq, sumo]
   codeowners:
     active: [sumo-drosiek, djaglowski]
+
+# TODO: Update the receiver to pass the tests
```

Comment:
Is there an issue to track this?

Reply:
Yes, https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27849